docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -88,7 +88,7 @@ The easiest way to convert is to use Teams Toolkit.
     :::image type="content" source="../images/teams-toolkit-create-outlook-task-pane-capability.png" alt-text="The two options in the App Features Using an Outlook Add-in drop down. The second option is called 'Import an Existing Outlook add-in'.":::
 
 1. In the **Existing add-in project folder** drop down, browse to the root folder of the add-in project.
-1. In the **Select import project manifest file** drop down, browse to the XML manifest file.
+1. In the **Select import project manifest file** drop down, browse to the add-in only manifest file, typically called manifest.xml.
```

Comment:
Elsewhere, we have a convention of putting filenames in bold. I think that should apply here too.
```suggestion
1. In the **Select import project manifest file** drop down, browse to the add-in only manifest file, typically named **manifest.xml**.
```